src/engraving/types/constants.h

```diff
@@ -30,4 +30,9 @@ struct Constant
 };
 }
 
+//! NOTE compat
+namespace Ms {
+using Constant = mu::engraving::Constant;
```

Comment:
Wouldn't it be nicer to call this struct `Constants`? That sounds more natural to me (like, you take the "division" from the (list of) Constants).

Reply:
ok, I agree